docs/index.md

```diff
@@ -20,6 +20,9 @@
 
 **TUI's Transfer Service** provides guests with rides between places during tours: from the airport to their hotel and back, to ports and back for cruises, and between hotels for multi-hotel vacations. Each one-way ride is a **Transfer**. Mobi's Continuous Planning Service enables TUI’s Transfer Service to operate efficiently by planning optimized trips in advance and enabling fast on-the-fly adjustments in response to disruptions like flight delays or vehicle breakdowns.
 
+Jamie: ^^^Obviously we want to phrase this in a customer-agnostic way...  Ideally the primary documentation would be
+universal, but for API details we could branch off into "version 1" and "version 2."
+
```

Comment:
Can we also take this opportunity to set terminology that we prefer? Primary example in my mind is that I prefer the term "Trip" over "Transfer"